src/html/erc20-form.html

```diff
@@ -112,17 +115,22 @@ <h3>Enter token address</h3>
     const amount = window.dom.find('erc20Amount')
     const select = window.dom.find('erc20SelectButton')
     const submit = window.dom.find('erc20Submit')
+    const cancel = document.getElementById('erc20Cancel')
 
     function disableForm () {
       amount.disabled = true
       select.disabled = true
       submit.disabled = true
+      cancel.disabled = true
+      window.dom.show('signInstruction')
```

Comment:
Can we only show this if you're using WalletConnect, so MetaMask folks don't get confused?

Reply:
Nevermind, seems like it probably makes sense either way